### PR TITLE
docs: Improve documentation for running integration tests

### PR DESCRIPTION
**Pull request by AlinoeDoctari**
Small improvement in the documentation for running integration test. Moved from #8850.

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -50,6 +50,8 @@ AWS_ACCESS_KEY_ID=XXX AWS_SECRET_ACCESS_KEY=xxx npm run integration-test-run-all
 
 _Note: Home folder is mocked for test run, therefore relying on `AWS_PROFILE` won't work. _ and _secret key_, need to be configured directly into env variables\_
 
+\_Note: Some integration tests depend on shared infrastructure stack (see below)\_
```
Comment (pgrzesik) [inline test/README.md:53]: If you'd like to apply formatting, I think you shouldn't escape the `_` here

**Review by pgrzesik — CHANGES_REQUESTED** [review]
Thanks @AlinoeDoctari, I believe the formatting is not applied correctly if that was the intention, please see my comment and let me know :raised_hands:

File: test/README.md (hunk 1 above)
Comment (AlinoeDoctari) [reply test/README.md:53]: Good point :sweat_smile: , I've not understood the formatting on the previous line.

Comment (pgrzesik) [reply test/README.md]: No worries, I think the previous line has broken formating :grimacing:

Comment (AlinoeDoctari) [reply test/README.md]: Formatting for the new line should be ok now :smile:

**Review by pgrzesik — APPROVED** [review]
thank you @AlinoeDoctari :bow:

**Pull request merged**